tests/e2e/tests/regression/browser/keys-all-databases.e2e.ts

```diff
@@ -0,0 +1,110 @@
+        await acceptLicenseTermsAndAddRECloudDatabase(cloudDatabaseConfig);
+    })
+    .after(async() => {
+        //Clear and delete database
+        await browserPage.deleteKeyByName(keyName);
+        await deleteDatabase(cloudDatabaseConfig.databaseName);
+    })('Verify that user can add Key in RE Cloud DB', async t => {
+        keyName = common.generateWord(10);
+        //add Hash key
+        await browserPage.addHashKey(keyName);
+        //check the notification message
+        const notification = await browserPage.getMessageText();
+        await t.expect(notification).contains('Key has been added', 'The notification');
+        //check that new key is displayed in the list
+        await browserPage.searchByKeyName(keyName);
+        const isKeyIsDisplayedInTheList = await browserPage.isKeyIsDisplayedInTheList(keyName);
+        await t.expect(isKeyIsDisplayedInTheList).ok('The key is added');
+    });
+test
+    .meta({ rte: rte.ossCluster })
+    .before(async() => {
+        await acceptLicenseTermsAndAddOSSClusterDatabase(ossClusterConfig, ossClusterConfig.ossClusterDatabaseName);
+    })
+    .after(async() => {
+        //Clear and delete database
+        await browserPage.deleteKeyByName(keyName);
+        await deleteOSSClusterDatabaseApi(ossClusterConfig);
+    })('Verify that user can add Key in OSS Cluster DB', async t => {
+        keyName = common.generateWord(10);
+        //add Hash key
+        await browserPage.addHashKey(keyName);
+        //check the notification message
+        const notification = await browserPage.getMessageText();
+        await t.expect(notification).contains('Key has been added', 'The notification');
+        //check that new key is displayed in the list
+        await browserPage.searchByKeyName(keyName);
+        const isKeyIsDisplayedInTheList = await browserPage.isKeyIsDisplayedInTheList(keyName);
+        await t.expect(isKeyIsDisplayedInTheList).ok('The key is added');
+    });
+test
+    .meta({ env: env.web, rte: rte.sentinel })
+    .before(async() => {
+        await acceptLicenseTermsAndAddSentinelDatabaseApi(ossSentinelConfig);
+    })
+    .after(async() => {
+        //Clear and delete database
+        await browserPage.deleteKeyByName(keyName);
+        await deleteSentinelDatabaseApi(ossSentinelConfig);
+    })('Verify that user can add Key in Sentinel Primary Group', async t => {
+        keyName = common.generateWord(10);
+        //add Hash key
+        await browserPage.addHashKey(keyName);
+        //check the notification message
+        const notification = await browserPage.getMessageText();
+        await t.expect(notification).contains('Key has been added', 'The notification');
+        //check that new key is displayed in the list
+        await browserPage.searchByKeyName(keyName);
+        const isKeyIsDisplayedInTheList = await browserPage.isKeyIsDisplayedInTheList(keyName);
+        await t.expect(isKeyIsDisplayedInTheList).ok('The key is added');
```

Comment:
Let's make a function like you already did in the other test

Reply:
yes, changed, thanks